usaspending_api/awards/management/commands/loadcontracts.py

```diff
@@ -26,25 +27,227 @@ def handle(self, *args, **options):
         field_map = {
             "federal_action_obligation": "dollarsobligated",
             "description": "descriptionofcontractrequirement",
+            "other_statutory_authority": "otherstatutoryauthority",
             "modification_number": "modnumber"
         }
 
         value_map = {
             "data_source": "USA",
-            "award": lambda row: Award.objects.get_or_create(piid=row['piid'], type='C')[0],
-            "recipient": lambda row: LegalEntity.objects.get_or_create(recipient_name=row['dunsnumber'])[0],
-            "awarding_agency": lambda row: Agency.objects.get(subtier_code=self.get_agency_code(row['maj_agency_cat'])),
+            "award": lambda row: self.create_or_get_award(row),
+            "awarding_agency": lambda row: self.get_agency(row),
```

Comment:
general question about all of the cleanup you're doing here: since award transaction data that we're getting from the broker is coming from the same source systems that feed the current usaspending, will we also need to apply the cleanup to the D1/D2 broker files?

Likely not something we need to worry about immediately, but just planting a seed, since the broker doesn't really validate D1/D2 details, and I'm sure where the data quality breakdown is occurring.